resources/usa_carshop/vehicle-shop_sv.lua

```diff
@@ -686,33 +693,198 @@ AddEventHandler("vehicle-shop:loadItems", function()
+				stolenVehicles[k] = nil
+			else
+				v.coords = GetEntityCoords(ent)
+				for i,source in ipairs(v.trackedBy) do
+					TriggerClientEvent("vehShop:trackStolenVeh", source, v.coords)
+				end
+			end
+		end
+	end
+end)
+
+TriggerEvent('es:addJobCommand', 'trackveh', {'sheriff', 'corrections'}, function(source, args, char)
+	local plate = nil
+	if args[2] ~= nil then
+		plate = string.upper(args[2])
+	else
+		TriggerClientEvent("usa:notify", source, "No plate provided using last 911")
+		plate = lastStolenPlate
+	end
+	if stolenVehicles[plate] ~= nil then
+		TriggerClientEvent("vehShop:trackStolenVeh", source, stolenVehicles[plate].coords)
+		table.insert(stolenVehicles[plate].trackedBy, source)
+	else
+		TriggerClientEvent("usa:notify", source, "Invalid Plate! Vehicle might be destroyed, lost or impounded!")
+	end
+end, {
+	help = "Track a stolen vehicle.",
+	params = {
+		{ name = "Plate", help = "The plate of the tracker fitted vehicle, if not provided will use plate from last 911" }
+	}
+})
+
+TriggerEvent('es:addJobCommand', 'endtrack', {'sheriff', 'corrections'}, function(source, args, char)
+	TriggerClientEvent("vehShop:stopTrackStolenVeh", source)
+	for k,v in pairs(stolenVehicles) do
+		for i,s in ipairs(v.trackedBy) do
+			if source == s then
+				table.remove(v.trackedBy, i)
+			end
+		end
+	end
+end, {
+	help = "Stops tracking a stolen vehicle.",
+	params = {
+	}
+})
+
+Citizen.CreateThread(function()
+	while true do
+		local lastCheck = GetGameTimer()
+		while GetGameTimer() - lastCheck < 5000 do
+			Wait(1)
+		end
+		for k,v in pairs(testDrivers) do
+			local ent = NetworkGetEntityFromNetworkId(v.netID)
+			local char = exports["usa-characters"]:GetCharacter(v.source)
+			if v.netID ~= nil and (not DoesEntityExist(ent) or GetEntityModel(ent) ~= v.hash) then
+				addStrike(char)
+				local strikes, bannedFor = getStrikes(char)
+				TriggerClientEvent("usa:notify", v.source, "Your loaned car was destroyed or lost! "..strikes.."/3 strikes!", "^3INFO: ^0Your loaned car was destroyed or lost! "..strikes.."/3 strikes!")
+				TriggerClientEvent("vehShop:endTestDrive",v.source)
+				testDrivers[k] = nil
+			else
+				local diff = GetGameTimer() - v.timestamp
+				if diff > warnMinutes * 60 * 1000 and testDrivers[k].warned == false then
+					testDrivers[k].warned = true
+					TriggerClientEvent("usa:notify", v.source, "You have driven the car for " .. tostring(warnMinutes) .. " minutes, return to the dealership immediatly!", "^3INFO: ^0You have driven the car for " .. tostring(warnMinutes) .. " minutes, return to the dealership immediatly!")
+				end
+				if diff > finalMinutes * 60 * 1000 then
+					exports.globals:getNumCops(function(numCops)
+						if numCops > 3 then
+							addStrike(char)
+							local strikes, bannedFor = getStrikes(char)
+				TriggerClientEvent("usa:notify", v.source, "Your loaned car was destroyed or lost! "..strikes.."/3 strikes!", "^3INFO: ^0Your loaned car was destroyed or lost! "..strikes.."/3 strikes!")
```

Comment:
Hm ok, so basically that will just determine whether the script deletes the vehicle or not after the time limit then...

So in that case this help message is wrong then? wouldn't make sense to say the loaned car was destroyed or lost if they still have it in their possession right
```suggestion
local msg = "You failed to return the vehicle! Police have been called! " .. strikes .. "/3 strikes!"
				TriggerClientEvent("usa:notify", v.source, msg, "^3INFO: ^0" .. msg)
```

Reply:
Yea that was weird i might have must pasted something wrong its fixed now with [c3a1dbe](https://github.com/minipunch/usarealismrp-fxserver/pull/1575/commits/c3a1dbe199e8360094f4de4cb59036e907bfa405)